src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -203,24 +203,24 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
   }
 
   constructor(router: Router,
-              instructorService: InstructorService,
```

Comment:
Could you add back the indentation for this file? This may have been introduced during your merge, [this article](https://dev.to/lofiandcode/git-and-github-how-to-revert-a-single-file-dha) might help, since it seems the changes you've made are easy to add back in (just the `scrollToTopOfPage`?)